pkg/ccl/backupccl/schedule_exec.go

```diff
@@ -275,7 +275,11 @@ func (e *scheduledBackupExecutor) GetCreateScheduleStatement(
 	}
 
 	node := &tree.ScheduledBackup{
-		ScheduleLabel:   tree.NewDString(sj.ScheduleLabel()),
+		// ScheduleLabelSpec.IfNotExists is always set to false here as this attribute is only relevant
+		// during the createBackupScheduleHook function which has already been called and will not
+		// get called again.
+		ScheduleLabelSpec: &tree.ScheduleLabelSpec{false,
```

Comment:
we usually label the field names of the struct which we are assigning values to:

ScheduleLabelSpec{IfNotExits: false, ...}

Reply:
done